tests/perf-system/submitter/handle_arguments.hpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
A few coding style changes to match the CCF coding style:
- File should be `handle_arguments.h` rather than `.hpp`
- `#ifndef HANDLE_ARGUMENTS_H` guards should be replaced with a single `#pragma once`
- Functions we define should be `snake_cased` rather than `camelCased` (ie - `read_parquet_file` rather than `readParquetFile`)